ports/stm32/boards/Passport/modules/flows/show_security_words_setting_flow.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2022 Foundation Devices, Inc. <hello@foundationdevices.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# show_security_words_setting_flow.py - Choose to decide to show security words at login or not.
+
+import lvgl as lv
+from flows import Flow
+from pages import PINEntryPage, ShowSecurityWordsSettingPage
+import microns
+
+
+class ShowSecurityWordsSettingFlow(Flow):
+    def __init__(self):
+        super().__init__(initial_state=self.select_setting, name='LoginFlow')
+
+    async def select_setting(self):
+        selected_value = await ShowSecurityWordsSettingPage().show()
+        if selected_value:
+            self.goto(self.enter_pin)
+        else:
+            self.set_result(None)
+
+    async def enter_pin(self):
```

Comment:
We might want an intermediate page here telling the user what is about to happen.  In other words, why are they about to be asked to enter their PIN, and what are these "Security Words" even for?

@zachherbert

Reply:
We discussed this offline and have a new approach.  New PR coming when that's implemented.

Reply:
I pushed what we discussed by checking the sha256 of the first 4 pin digits. My only concern is using an extra 32 bytes of RAM for the volatile setting.